### ISSUE TITLE
Update NTIA Conformance Checker version

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-online-tools#418]
There's been quite a few improvements to the NTIA conformance checker.  We could just pull the latest or wait for an official release (see  https://github.com/spdx/ntia-conformance-checker/issues/22 for current status of the NTIA conformance checker release process).

**Comment by goneall** [comment]
When upgrading to version 0.1.0 I'm getting the following import error:

```
spdx_dev  |   File "/spdxonlinetools/src/src/urls.py", line 37, in <module>
spdx_dev  |     from api import views
spdx_dev  |   File "/spdxonlinetools/src/api/views.py", line 39, in <module>
spdx_dev  |     import app.core as core
spdx_dev  |   File "/spdxonlinetools/src/app/core.py", line 19, in <module>
spdx_dev  |     from ntia_conformance_checker.cli_tools import check_anything
spdx_dev  |   File "/src/ntia-conformance-checker/ntia_conformance_checker/cli_tools/check_anything.py", line 7, in <module>
spdx_dev  |     import spdx.creationinfo
spdx_dev  | ModuleNotFoundError: No module named 'spdx'
```

@jspeed-meyers @linynjosh - can you help resolve this?

I created a draft PR #419 with the changes

**Comment by jspeed-meyers** [comment]
Hmm, the first clue is the `ModuleNotFoundError: No module named 'spdx'` line. The `spdx` refers to the `tools-python` library. For some reason, that module can't be found.

Here's my prediction about a fix (and apologies in advance for the Python minutiae):

- `pip` needs to install a pre-release version of `https://pypi.org/project/spdx-tools/`. That's because `ntia-conformance-checker` nows depends on that latest pre-release so that it doesn't need to rely on either the latest non-pre-release version of `tools-python` from 2019 or Josh's fork).
- But `pip`, by default, won't install pre-release versions
- To enable this installation, all `pip install`'s need a `--pre` flag to be added. (According to my [scan of the repo](https://github.com/spdx/spdx-online-tools/search?q=pip), that means updates to the `django.yml`, `dockerfile.dev`, and `dockerfile.prod`)

If that doesn't work, I'm glad to keep debugging. Hopefully this `--pre` issue will go away soon :)

**Comment by jspeed-meyers** [comment]
A plan B is to follow the plan above and to explicitly add `spdx-tools==0.7.0rc0` to the `requirements.txt` file in this project.

**Comment by goneall** [comment]
Thanks @jspeed-meyers - Plan B worked.

**Comment by goneall** [comment]
Resolved with PR #419
